gradle.properties

```diff
@@ -1,3 +1,4 @@
 org.gradle.caching=true
 group=org.ballerinalang.googleapis.people
-version=2.1.0
+version=2.2.0
+ballerinaLangVersion=2201.0.0
```

Comment:
Please remove this ballerinaLangVersion=2201.0.0

Reply:
Removed as suggested.